pages/pipelines/defining_steps.md.erb

```diff
@@ -8,27 +8,29 @@ Defining your pipeline steps in a `pipeline.yml` file gives you access to more c
 
 ## Getting started
 
-After creating a new pipeline, click the ➕ to add a step and choose 'Read steps from repository'. On the backend, this type of step will run `buildkite-agent pipeline upload`.
+In your pipeline settings, click the ➕ to add a step and choose “Read steps from repository”:
 
-<%= image("pipeline-upload-step.png", width: 612/2, height: 484/2) %>
+<%= image "pipeline-upload-step.png", width: 612/2, height: 484/2, alt: "Screenshot of the 'Read steps from repository' menu item" %>
 
-When this step runs on an agent it reads in the pipeline file in your repository from `.buildkite/pipeline.yml`. 
+Your pipeline steps will now show a new :pipeline: step that runs a `buildkite-agent pipeline upload` command:
 
-For example, here is a pipeline with one step that prints “Hello!” to your build log:
+<%= image "pipeline-upload.png", width: 1174/2, height: 452/2, alt: "Screenshot of the pipeline upload step" %>
+
+When this step runs on an agent, it reads in the pipeline file from `.buildkite/pipeline.yml` in your repository. For example, here is a pipeline file with a single step that prints “Hello!” to the build log:
 
 ```yml
 steps:
   - label: Example Test
     command: echo "Hello!"
 ```
 
-To test your new `.buildkite/pipeline.yml` simply commit the file, push the commit, and run a new build. After it has run, it will turn green and look similar to this:
+To test your `.buildkite/pipeline.yml` file, commit and push it to your repository, and (if it didn't happen automatically) run a new build:
```

Comment:
Hrmm good question. We should probably assume people actually setup the webhook (in a getting started guide), because the UI tells you do so when you create the pipeline.